Providers/Profile.swift

```diff
@@ -1051,6 +1097,9 @@ open class BrowserProfile: Profile {
                 return deferMaybe(statuses)
             }
 
+            // TODO: Invoke `account.commandsClient.fetchMissedRemoteCommands()` to
+            // catch any missed FxA commands at time of Sync.
+
```

Comment:
I'm not sure we should have this here specifically; this function is deep in the weeds of sync coordination. 

Suggest, however that we poll for missing remote commands in `applicationDidBecomeActive()` if push has been enabled, and `syncEverything()` is not.